src/cli/cli.rs

```diff
@@ -1,13 +1,27 @@
 use configuration::Settings;
 use clap::Parser;
+use std::io::prelude::*;
+use std::fs::File;
+use std::path::Path;
 
 fn main() -> Result<(), serde::yaml::Error> {
```

Comment:
you can have only one main function in the project, which is actually implemented in  `./src/main.rs`